stacks/openshift/application/oidc-authservice-for-appid/README.md

```diff
@@ -0,0 +1,19 @@
+# oidc-authservice for IBM Cloud AppID
```

Comment:
General: if there is a document in kubeflow.org, we should just point to that instead of separate instructions.

Reply:
I did find [Authentication using OIDC](https://www.kubeflow.org/docs/aws/authentication-oidc/) but it uses Auth0 instead of AppID. in this case, no, no articles in kubeflow.org yet.

Reply:
Sorry I should have been clear. I meant to say that if you are planning to write an article under the [Kubeflow on IBM Cloud](https://www.kubeflow.org/docs/ibm/) section, the readme here should just point to that link instead. Also, we will need a doc in the KubeflowDojo. Thought is that a readme inside the manifests won't catch user's attention as it is not actually exposed to users who use the manifests.